files/.gitignore

```diff
@@ -0,0 +1,3 @@
+*
+!.gitignore
+
```

Comment:
We have already have a folder `upload/` containing uploaded documents in `upload/documents/`. Consider using the same directory.

I like the idea of adding this file tho as it means we don't manually have to create the destination folders (which i have already forgotten on multiple occasions) for documents/templates!